chain/chain/src/lightclient.rs

```diff
@@ -67,26 +57,26 @@ pub fn create_light_client_block_view(
         .map(|(i, v): (usize, &ValidatorStakeView)| (v.account_id.clone(), i))
         .collect::<HashMap<_, _>>();
 
-    let mut approvals_next = vec![None; validator_ords.len()];
-    let mut approvals_after_next = vec![None; validator_ords.len()];
+    let /*mut*/ approvals_next = vec![None; validator_ords.len()];
+    let /*mut*/ approvals_after_next = vec![None; validator_ords.len()];
 
     let next_block_hash = chain_store.get_next_block_hash(&block_header.hash())?.clone();
-    let next_block_header = chain_store.get_block_header(&next_block_hash)?;
+    //let next_block_header = chain_store.get_block_header(&next_block_hash)?;
```

Comment:
We can remove it, but I suspect that parts of it will be needed when we implement the new light client.

(I can still remove it and then just fetch whatever is needed from the git history)